### PR TITLE
feat: update balanced tvl

### PR DESCRIPTION
**Pull request by hetfly**
PR contains:

updated collateral tvl calculation,
added new pools for dex tvl calculation,
added stability fund tvl

### CODE REVIEW

File: projects/balanced/pools.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        name: 'CROWN/bnUSD',
+        baseCurrencyKey: 'CROWN',
+        quoteCurrencyKey: 'bnUSD',
+    },
+    {
+        chainId: 1,
+        id: 50,
+        name: 'CLAW/sICX',
+        baseCurrencyKey: 'CLAW',
+        quoteCurrencyKey: 'sICX',
```
Comment (ajaya-man) [inline projects/balanced/pools.js:112]: CROWN/bnUSD and CLAW/sICX seems to have same id.

Comment (hetfly) [reply projects/balanced/pools.js]: Nice spotting, fixed, thanks!

**Comment by llamatester** [comment]
The adapter at projects/balanced exports TVL: 
        
 
 
    
    total                    16.92 M

**Comment by realdealshaman** [comment]
@hetfly Hi, we actually stopped counting the stability pool in Liquity and all Liquity forks, I think we would hve to apply the same here

**Comment by BennyOptions** [comment]
Hey @realdealshaman, this is actually not a Liquity fork. It's written in Java and on the ICON Blockchain. Balanced launched in April of 2021. I don't know Liquity well, so not sure what the stability pool means in their context. I know it's a stablecoin backed only by ETH, so I suspect it's likely a different functionality and coincidentally has a similar name. Happy to elaborate on our stability fund functionality if you'd like.

**Comment by realdealshaman** [comment]
Got it thanks for the info regarding your stability fund

**Pull request merged**